# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1308926230 14400
#      Fri Jun 24 10:37:10 2011 -0400
# Branch stable
# Node ID 37c5cf08cfa09b788f91ed7afbeb1836a3a747df
# Parent  3666e8e6f96e6899b8306d6ea9614aadf0500d67
Added tag release-3-4-2 for changeset 3666e8e6f96e

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -46,8 +46,9 @@ a1d20052517a950033896f5a876de8f3ee520138
 74be4b7273e43952e1afcd9ad979d35667c7f82e ss-3-3-50
 79a56d0a6a0d693ebf461c456951b20dfbcde140 ss-3-3-51
 229675bb7647f7e7e499bcb5acacc8eabe7f3697 ss-3-3-52
 6c69a7c39039bccd5a2fe481fcc7f896c82b160b ss-3-3-53
 3cbc0d77db48aec32bcb202d09a036d2cb9cc3b9 ss-3-3-53
 bd2643f0ce57d070963bedd48857405f6924aa85 ss-3-3-54
 695141f1c05cf1b240592bdd18e7a1503bb2a539 ss-3-3-55
 901d466ee55ac902a875ec0ade6f1eccef0841dc release-3-4-1
+3666e8e6f96e6899b8306d6ea9614aadf0500d67 release-3-4-2
